review08/README.md

```diff
@@ -0,0 +1,241 @@
+# 08 - Health, metrics, and monitoring
+
+## Health checks
+
+Health checks are critical for cloud applications to monitor the state of the application. They are used to verify the application's state. This usually means if the application is running, if it is ready to serve requests, or if it is connected to the database, Kafka, or other dependent services.
+
+For example, Kubernetes uses health checks to determine if a container is running and if it is ready to serve requests or if it should be restarted.
+
+It's also useful for monitoring tools to check the state of the application and send alerts if something is wrong and you need to take action.
+
+### What should be checked?
+
+- If the service is running.
+- If the database is connected.
+- If other dependencies are connected.
+- Check if external services are available.
+- Resource usage (CPU, memory, network,...).
+
+#### Types
+
+1. Liveness -- The liveness probe checks if the service is healthy. If the probe fails, the service is not running, cannot be recovered, and should be restarted.
+2. Readiness -- The readiness probe checks if the application is ready to serve traffic; requests are not prohibited. Dependencies, databases, and external services are connected. If the readiness probe fails, the service is not yet ready to serve traffic, but it's not considered a failure, and the service is not restarted. This probe prevents routing to the instance which is warming up or loading data.
```

Comment:
The first sentence doesn't make sense. probably just style, don't use ;

Reply:
why not yet? readiness can fail during the app lifecycle, DB connection can go down any time